database/mssql/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/mssql/rhel/server:2019-CU1-rhel-8
+
```

Comment:
@Fosol I think we should use the `2019-latest` tag to match our Openshift environment

Reply:
Fixed